mne/epochs.py

```diff
@@ -1556,13 +1556,10 @@ def equalize_event_counts(self, event_ids, method='mintime', copy=True):
             If 'truncate', events will be truncated from the end of each event
             list. If 'mintime', timing differences between each event list
             will be minimized.
-        copy : bool
-            If True, a copy of epochs will be returned. Otherwise, the
-            function will operate in-place.
```

Comment:
Hm ... that seems to have been the prior behavior. @teonlamont what actually should we do here? Because in other places, what we do is:

```
inst = self.copy() if copy else self
...  # do stuff with inst
return inst
```

E.g., that is what `epochs.resample` does, and it is what `equalize_event_counts` used to do before. And it seems to me that this was consistent.